src/runtime/mini_hexagon_dma.h

```diff
@@ -22,7 +22,7 @@ __inline static int align(int x, int a) {
 
```

Comment:
It's possible the contents of this file were copy-pasted from elsewhere and should be kept unchanged to minimize diffs against the source. I'll tag a QC person: @pranavb-ca

Reply:
Thanks @abadams. This change is fine. We'll fold it into our sources to keep this maintainable in-house.